templates/artist/artist_detail.html

```diff
@@ -171,11 +184,18 @@ <h1 class="header-font">{{ artist.name }}</h1>
             <hr>
         </div>
         <!-- Current Investors -->
-        <h2>Current Investors</h2>
-        <ul>
+        <ul class="vertical menu" data-accordion-menu>
+          <li>
+        <a href= "#"><h2>CURRENT INVESTORS</h2></a>
+            <ul class="menu vertical nested is-active">
             {% for investor in investors %}
-                <li>{% if investor.public_profile_url %}<a href="{{ investor.public_profile_url }}"><img src="{{ investor.avatar_url }}" alt="{{ investor.name }}" /> {{ investor.name }}</a>{% else %}{{ investor.name }}{% endif %}: ${{ investor.total_investment }}</li>
+                <li>{% if investor.public_profile_url %}
```

Comment:
Oh true. Yeah, we don't want that spacing to affect anything. I didn't try this myself, but try adding the style `display: inline-block;` to the `<li>` tag:
```
<li style="display: inline-block;">
    {% if investor.public_profile_url %}
        ...
    {% endif %}
    : ${{ investor.total_investment }}
</li>

If that works, we should add `display: inline-block;` to one of the `.css` files.